articles/api-auth/token-introspection.md

```diff
@@ -0,0 +1,81 @@
+---
+description: How to use token introspection to get information about a token's authorization context
+toc: true
+---
+
+# Token Introspection
+
+Auth0 supports [OAuth 2.0 Token Introspection](https://tools.ietf.org/html/rfc7662), which allows you to ask Auth0 for details about any access tokens presented to your API. In addition to asking Auth0 about the *state* of the access token (whether it is active/expired/revoked), you can request information about the token's scopes and the context in which it was issued (such as who issued the token and to whom the token was issued).
+
+## Create an API Record with Auth0
+
+To use token introspection, you must set up an [API](/apis) that represents your protected resource(s).
+
+For assistance, please see [How to Configure an API in Auth0](/apis#how-to-configure-an-api-in-auth0). Alternatively, you can do this by making the appropriate `POST` call to the Management API's [Create a Resource Server endpoint](/api/management/v2#!/Resource_Servers/post_resource_servers).
+
+### Provide the Public Key to Auth0
+
+Because token introspection requires Auth0 to validate the signature of your access token, you'll need to create a key pair for your API and update your API's Auth0 record with the public key.
+
+There are three ways by which Auth0 can obtain the public key:
+
+1. You can define the `verificationKey` on the API's Auth0 record. This is a JSON Web Key (JWK) or PEM-encoded certificate.
+2. You can define the `verificationLocation` on the API's Auth0 record. This is a URI from which Auth0 fetches JSON Web Keys (JWK).
+3. If the API's `identifier` is `http:`- or `https:`-based (such as `audience=https://example.com/foo`), then Auth0 attempts to fetch the key from the well-known JWK registry of the host (for the provided URI example, Auth0 attempts to fetch from `https://example.com/.well-known/jwks.json`).
```

Comment:
in this case it wouldn't be audience=https:, it's referring to the identifier. 

So simply (such as `https://example.com/foo`)